DonjinKrawler-Client/src/main/java/donjinkrawler/visitor/ItemVisitorImpl.java

```diff
@@ -0,0 +1,62 @@
+package donjinkrawler.visitor;
+
+import donjinkrawler.Inventory;
+import donjinkrawler.items.*;
+import krawlercommon.PlayerData;
+import krawlercommon.composite.FinalBonus;
+import krawlercommon.composite.RawBonus;
+
+public class ItemVisitorImpl implements ItemVisitor {
```

Comment:
From the [checklist](https://dzone.com/articles/java-code-review-checklist):
`Make class final if not being used for inheritance.`